scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -135,10 +139,23 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
             ctx.replaceToken(close, ")")).getOrElse(Patch.empty)
     }.asPatch
 
+  def replaceTupleZipped(ctx: RuleCtx) =
+    ctx.tree.collect {
+      case tupleZipped(t @ Term.Select(Term.Tuple(args), _)) =>
+        val argTokens = args.flatMap(_.tokens).toSet
+        val removeTokensPatch = t.tokens.filter(!argTokens.contains(_)).map(ctx.removeToken).asPatch
```

Comment:
`nonTupleElementTokens`

`t.tokens.filterNot(tupleElementTokens)`

Reply:
PS, you might be removing comments here, I would keep comments

Reply:
The only tokens that need to be removed a leading `(`, trailing `)`, commas `,` and `.zipped`.

Reply:
yeah, you are right. Everything that is not a tuple element gets removed.

Preserving comments + trivia makes the rewrite a lot more complicated :smile:. For instance:
`(((xs, ys) /* foo1 */ zipped) /* foo2 */)` 

would ideally be rewritten to... 

`((xs.lazyZip(ys) /* foo1 */ ) /* foo2 */)`

Note that in this case the leading `(` and trailing `)` are not the outmost ones. It is totally doable (in fact I got it working already) but requires some extra steps to figure out the correct parenthesis and deal with the potentially missing `dot`.

Also, I think it would be nice to get rid of empty spaces between the comma and the first non-empty space token that follows it. Otherwise the output will be something like: 
`xs.lazyZip( ys)`